### PR TITLE
Backup flow

### LINKED ISSUE #3415: Create  a new flow to check that the user has copied the work backup correctly.

### PR DESCRIPTION
**Pull request by tanojaja**
Fixes #3415

### CODE REVIEW

File: src/js/controllers/backup.js [hunk 1]
```diff
@@ -1,69 +1,125 @@
...
+    function init() {
+      $scope.passphrase = '';
+      resetAllButtons();
+      customWords = [];
+      mnemonic = null;
+      self.xPrivKey = null;
+      self.step1 = true;
+      self.step2 = false;
+      self.step3 = false;
+      self.step4 = false;
```
Comment (isocolsky) [inline src/js/controllers/backup.js]: Please, use a single numeric variable `self.step` and redefine the `goToStepN()` function to accept `N` as an argument.

**Comment by tanojaja** [comment]
Update

**Comment by cmgustavo** [comment]
I found an issue: 

I already made a backup and also deleted the wallet seed, but I still can see the words. 
Also following the wizard, I enter the words in the right order, it always fails

**Comment by tanojaja** [comment]
Updated

**Comment by cmgustavo** [comment]
ACK

**Comment by matiu** [comment]
Great job!!

**Pull request merged**